_docs/community.md

```diff
@@ -54,76 +54,73 @@ which will produce something like the following...
 
 If you're looking for cost vs power using ApexCharts, then @plandregan has you covered
 
-```yaml
 type: custom:apexcharts-card
 experimental:
-color_threshold: true
+  color_threshold: true
 header:
-show: true
-floating: true
-title: Cost vs Power
+  show: true
+  floating: true
+  title: Cost vs Power
 graph_span: 24h
 show:
-last_updated: true
-loading: true
+  last_updated: true
+  loading: true
 apex_config:
-legend:
-show: false
-chart:
-zoom:
-type: x
-enabled: true
-autoScaleYaxis: false
-toolbar:
-show: true
-autoSelected: zoom
-xaxis.type: datetime
-fill:
-type: gradient
-gradient:
-shadeIntensity: 0.1
-opacityFrom: 0.3
-opacityTo: 1
-inverseColors: true
+  legend:
+    show: false
+  chart:
+    zoom:
+      type: x
+      enabled: true
+      autoScaleYaxis: false
+    toolbar:
+      show: true
+      autoSelected: zoom
+    xaxis.type: datetime
+  fill:
+    type: gradient
+    gradient:
+      shadeIntensity: 0.1
+      opacityFrom: 0.3
+      opacityTo: 1
+      inverseColors: true
 series:
-
-    entity: sensor.octopus_energy_electricity_***************************_current_rate
+  - entity: sensor.octopus_energy_electricity_xxxxxxxxxxxx_1100000946808_current_rate
```

Comment:
Are you ok with this serial number to appear?

Reply:
thanks for picking that up. xxxx it out if you don't mind